tests/base/src/main/resources/schema/elytron-client-1_4.xsd

```diff
@@ -487,10 +488,25 @@
                 </xsd:documentation>
             </xsd:annotation>
         </xsd:attribute>
-        <xsd:attribute name="maximum-cert-path" type="xsd:positiveInteger" use="optional">
```

Comment:
For backwards compatibility, I don't think we can remove the ```maximum-cert-path``` attribute here right away. I think a description should be added that indicates that it has been deprecated.

Reply:
fixed :)